src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java

```diff
@@ -13,6 +17,7 @@
 	public static final double TURN_EXP = 2;
 	public static final double Y_SPEED_SCALE = 1;
 	public static final double TURN_SPEED_SCALE = 1;
+	private final GearAlign gearAlign = new GearAlign(new AligningCameraGRIP());
```

Comment:
This is no different from just instantiating the object within the bindCommands method in terms of subclass accessibility.